lcov.info

```diff
@@ -1,312 +1,313 @@
 TN:
```

Comment:
This shouldn't be committed

✂️

Reply:
The repo is not configured properly but we need to display the coverage in PRs: https://github.com/zgosalvez/github-actions-report-lcov